File: fizzbuzz.cpp
#include <iostream>
using namespace std;

int main(){

	int number;
	for (number = 1; number < 101; number++){
		if (number%3!=0 && number%5!=0){
            cout<<number;
            cout<<"\n";
		}
		else if (number%3==0 && number%5!=0){
            cout<<"Fizz ";
            cout<<"\n";
		}
		else if (number%5==0 && number%3!=0){
            cout<<"Buzz ";
            cout<<"\n";
 		}
        else if (number%3==0 && number%5==0){
            cout<<"FizzBuzz ";
            cout<<"\n";
        	}
	}
return 0;
}

File: fizzbuzz.java
class FizzBuzz {

    public static void main(String args[]){

    int n;

        for (n = 1; n < 101; n++){

            if (n%3!=0 && n%5!=0){
                    System.out.print(n);
                    System.out.println();}
            else if (n%3==0 && n%5!=0){
                    System.out.print("Fizz \n");}
            else if (n%5==0 && n%3!=0){
                    System.out.print("Buzz \n");}
            else if (n%3==0 && n%5==0){
                    System.out.print("FizzBuzz \n");}
            else{
                    System.out.print("Error. \n");}
        }
    }
}
